nucypher/characters/lawful.py

```diff
@@ -12,38 +12,38 @@
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
-along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+along with nucypher.  If not, see <https://ww"Unable to glean address from node that purported to be version {}.  We're only version {}"w.gnu.org/licenses/>.
```

Comment:
Wait.. what happened here?

Reply:
Bad paste; my fault.  I'll push up fixes for some of the cruft soon.